File: vue-vben-admin/apps/web-naive/src/api/entity/SysSnapShot.ts
export interface SysSnapShot {
  type: string  | null | undefined ;    //  数据类型
  content: string  | null | undefined ;    //  数据值
  hash: string  | null | undefined ;    //  数据hash值
  businessId: string  | null | undefined ;    //  业务Id
  version: number  | null | undefined ;    //  版本
  tenantId: string  | null | undefined ;    //  租户GUID
  id: string  | null | undefined ;    //  主键
  createdId: string  | null | undefined ;    //  创建人GUID
  createdTime: string  | null | undefined ;    //  创建时间
  creator: string  | null | undefined ;    //  创建人
  updatedTime: string  | null | undefined ;    //  更新时间
  updatedId: string  | null | undefined ;    //  更新人GUID
  updator: string  | null | undefined ;    //  更新人
}

File: src/SimpleApp/Simple.AdminApplication/SysMng/Entities/SysSnapShot.cs
using System.ComponentModel.DataAnnotations;

namespace Simple.AdminApplication.SysMng.Entities
{
    /// <summary>
    /// 快照服务
    /// </summary>
    public class SysSnapShot : DefaultTenantEntity
    {
        /// <summary>
        /// 数据类型
        /// </summary>
        public string Type { get; set; }

        /// <summary>
        /// 数据值
        /// </summary>
        [StringLength(-1)]
        public string Content { get; set; }

        /// <summary>
        /// 数据hash值
        /// </summary>
        public string Hash { get; set; }

        /// <summary>
        /// 业务Id
        /// </summary>
        public Guid? BusinessId { get; set; }

        /// <summary>
        /// 版本
        /// </summary>
        public int Version { get; set; } = 1;
    }
}